pennylane_lightning/core/src/simulators/lightning_tensor/tncuda/gates/tests/CMakeLists.txt

```
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.20)
```

3.20 or 3.21 for cuQuantum based devices? 
```suggestion
cmake_minimum_required(VERSION 3.21)
```

---

Not required for `cutensornet` backend so far since there is no `.cu` file in the `Lightning.Tensor` backend. However, it's required for `Lightning.GPU` due to the `initSV.cu`.